### PR TITLE
Add Firechain Mainnet & Rinia Testnet

### PR DESCRIPTION
**Pull request by 7bp**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
explorers do not seem to work now - please remove them for now

**Comment by 7bp** [comment]
@ligi Thank you, I removed the explorers for now.

Danke.

**Review by ligi — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

also: short name must be unique - but found `FIRE` more than once

**Comment by 7bp** [comment]
I applied the prettier formatting, also changed the short name for the testnet. Thank you!

File: _data/icons/rinia.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ",
+    "width": 512,
+    "height": 512,
```
Comment (ligi) [inline _data/icons/rinia.json:5]: identify QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ         12:47:05
QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ PNG 2000x2000 2000x2000+0+0 8-bit sRGB 653045B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/rinia.json (hunk 1 above)
Comment (7bp) [reply _data/icons/rinia.json:5]: Updated the icon to reflect the given size

**Pull request merged**